subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -34,7 +34,7 @@ Some plugins will break with this new version of Gradle, for example because the
 . Try to run the project and debug any errors using the <<troubleshooting.adoc#troubleshooting, Troubleshooting Guide>>.
 
 [[changes_7.0]]
-== Upgrading from 6.8
+== Upgrading from 6.8 or 6.9
```

Comment:
Now having doubts with what I suggested.

By default our sections say "Upgrading from 6.8" which means upgrading Gradle 6.8 to the next one, which is implied 7.0 back when this was created - and as indicated by the id.

Now that we created a 6.9, what we really should reflect is that:
* Upgrading from 6.8 (to implied 6.9) is a no-op, nothing needs to be done
* Upgrading from 6.9 (to implied 7.0) requires all the content under the current 6.8 entry

If it was only the second line, we could merge both as done here.
But thinking more about it, maybe we should really create and empty Upgrading from 6.8 (with ID changes_6.9) and move the content to a new Upgrading from 6.9 - which would keep `the changes_7.0` ID

Reply:
I think having an explicit title per version saying something like "nothing to do for this one" would be better 👍 

Your last proposal makes sense Louis 👍